diplo.lua

```diff
@@ -0,0 +1,88 @@
+--[[
+diplo - Quick "Diplomacy"
+Without arguments:
+    Print list of civs the player civ has diplomatic relations to.
+    This list might not match the ingame civ list, but is what matters. Does not include 'no contact' civs, etc.
+    Also shows if relations match between player civ and other civ - a mismatch can potentially happen 'naturally', but is important when trying to make peace proper, as both civs need to be at peace with the other.
+With arguments:
+    diplo CIV_ID RELATION
+    Changes the relation to the civ identified by CIV_ID to the one specified in RELATION, making sure that this is mutual.
+    Relations: 0 = Peace; 1 = War; 3 = Alliance (seems to only apply to site govs...?)
+    Civ list is shown afterwards.
+]]
+
+local args = {...}
+
+-- player civ references:
+local p_civ_id = df.global.plotinfo.civ_id
+local p_civ = df.historical_entity.find(df.global.plotinfo.civ_id)
+
+-- get a very rough, but readable name for a civ:
+function get_raw_name(civ)
+    local raw_name = ""
+    for _, name_word in pairs(civ.name.words) do
+        if name_word ~= -1 then
+            raw_name = raw_name .. " " .. df.global.world.raws.language.words[name_word].word
+        end
+    end
+    raw_name = string.sub(raw_name, 2)
+    return raw_name
+end
+
+-- if no civ ID is entered, just output list of civs:
+if not args[1] then
+    goto outputlist
+end
+
+-- make sure that there is a relation to change to:
+if not args[2] then qerror("Missing relation!") end

```

Comment:
you can output the script help here (in addition to the `qerror`) via `print(dfhack.script_help())`

Reply:
I'll add that. Do I need to somehoe 'import' utils for that to work...?

Reply:
it's a global dfhack function. nothing to import

Reply:
Added it.